sdcm/cluster_k8s.py

```diff
@@ -107,9 +107,10 @@ def setup_minikube_ubuntu18(node: cluster.BaseNode, kubectl_version: str, miniku
             chmod +x /usr/local/bin/minikube
 
             # Install Helm 3.
-            snap install helm --classic
+            snap install helm --channel=3.2/stable --classic
             ln -s /snap/bin/helm /usr/local/bin/helm
-        """)
+            chmod +x /usr/local/bin/helm
+            """)
```

Comment:
Don't think we need these changes.

Reply:
@dkropachev why did you add the `chmod` line?